src/components/DocExplorer.js

```diff
@@ -103,12 +114,21 @@ export class DocExplorer extends React.Component {
           field={typeOrField}
           onClickType={this._onClickTypeOrField}
         />;
+    } else if (this.state.searchValue) {
+      title = 'Search Results';
+      content = <SearchDoc
```

Comment:
nit: use the same styling here as in the else block below, line break and indent.